.gitmodules

```diff
@@ -2,3 +2,5 @@
 	path = ethereum/referencetests/src/test/resources
 	url = https://github.com/ethereum/tests.git
 	ignore = all
+	branch = 48142b4ee7a42d081c6eda7b2a57d9b2c958bc81
+	shallow = true
```

Comment:
A history depth of one for the Reference tests is a good change 👍

Reply:
Good change... for another PR.  Being a gigantic PR all changes should serve the purpose of integrating the Tuweni library.

Reply:
I had to make that change to force the build to pass.

Reply:
This incantation should get the submodule to match the head of besu
```
pushd ethereum/referencetests/src/test/resources/
git fetch origin develop
git reset 73bfbcc213efd7cabe678a46c83596024e85e2e2
popd
```

Reply:
I am reverting the commit, will open another PR for it.

Reply:
That broke the build, probably due to caching of the source repository in earlier steps. I can't force updating the reference tests submodule since ignore is set to all, and this commit is now reverted.

Reply:
I shouldn't write shells scripts after 11 at night, I forgot the `cd ..; git add resources/` before the popd.